config/services.yaml

```diff
@@ -24,3 +24,22 @@ services:
 
     # add more service definitions when explicit configuration is needed
     # please note that last definitions always *replace* previous ones
+    App\Security\Authenticator\LoginFormAuthenticator:
+        arguments:
+            - '@Symfony\Component\Routing\Generator\UrlGeneratorInterface'
+            - '@App\Repository\ContactRepository'
+
+    App\Controller\Security\SamlBundle\SamlController:
+        arguments:
+            - '@logger'
+            - '@surfnet_saml.metadata_factory'
+            - '@surfnet_saml.http.post_binding'
+            - '@surfnet_saml.remote.idp'
+            - '@surfnet_saml.hosted.service_provider'
+
+    # Firewall
+    surfnet_saml.saml_provider:
+        class: App\Security\SamlBundle\Authentication\SamlProvider
+
+    App\Security\SamlBundle\Authentication\SamlProvider:
+        alias: surfnet_saml.saml_provider
```

Comment:
Ik gok dat dit iets korter kan: `surfnet_saml.saml_provider: '@App\Security\SamlBundle\Authentication\SamlProvider`.

Overigens zou ik er niet voor kiezen om een "SamlBundle" namespace te introduceren, dat is wellicht verwarrend omdat je hier niet een Bundle aan het bouwen bent. Ik zou dat stukje gewoon weglaten denk ik.